BUILDIT.md

```diff
@@ -1628,31 +1619,26 @@ defmodule AppWeb.AppLive do
   def handle_event("update-item", %{"id" => item_id, "text" => text}, socket) do
     current_item = Item.get_item!(item_id)
     Item.update_item(current_item, %{text: text})
-    socket = assign_socket(socket)
-    AppWeb.Endpoint.broadcast_from(self(), @topic, "update", socket.assigns)
-    {:noreply, socket}
-  end
 
-  @impl true
```

Comment:
The `handle_info` for `start|stop`, `delete` and `update` events are the same. So only keeping `update` which will get the list of items each time an event occurs on the list (add item, delete item, toggle and archive)